src/soep_preparation/clean_variables/pequiv.py

```diff
@@ -77,6 +52,104 @@ def clean(raw_data: pd.DataFrame) -> pd.DataFrame:
     out["federal_state_of_residence"] = object_to_str_categorical(
         series=raw_data["l11101"], ordered=False
     )
+
+    # hh social benefits
+    out["arbeitslosengeld_2_y_hh_pequiv"] = object_to_int(raw_data["alg2"])
+    out["arbeitslosengeld_2_m_hh_pequiv"] = apply_smallest_float_dtype(
+        out["arbeitslosengeld_2_y_hh_pequiv"] / 12
+    )
+    out["mutterschaftsgeld_y"] = object_to_int(raw_data["imaty"])
+    out["kindergeld_y_hh_pequiv"] = object_to_int(raw_data["chspt"])
+    out["kindergeld_m_hh_pequiv"] = apply_smallest_float_dtype(
+        out["kindergeld_y_hh_pequiv"] / 12
+    )
+    out["kinderzuschlag_y_hh_pequiv"] = object_to_int(raw_data["adchb"])
+    out["kinderzuschlag_m_hh_pequiv"] = apply_smallest_float_dtype(
+        out["kinderzuschlag_y_hh_pequiv"] / 12
+    )
+    # betreuungsgeld only available 2014 through 2016
+    out["betreuungsgeld_y_hh"] = object_to_int(raw_data["chsub"])
+    out["wohngeld_y_hh_pequiv"] = object_to_int(raw_data["house"])
+    out["wohngeld_m_hh_pequiv"] = apply_smallest_float_dtype(
+        out["wohngeld_y_hh_pequiv"] / 12
+    )
+    # eigenheimzulage only available 1996 through 2014
+    out["eigenheimzulage_y_hh"] = object_to_int(raw_data["hsup"])
+
+    out["allgemeine_sozialhilfe_y_hh"] = object_to_int(raw_data["subst"])
+    out["grundsicherung_im_alter_y_hh"] = object_to_int(raw_data["ssold"])
+    out["pflegegeld_y_hh"] = object_to_int(raw_data["nursh"])
+    # sonstige sozialhilfe available in 1984 through 1991 and 2001 through 2009
+    out["sonstige_sozialhilfe_y_hh"] = object_to_int(raw_data["sphlp"])
+
+    # individual social benefits
+    out["arbeitslosengeld_y"] = object_to_int(raw_data["iunby"])
+    # arbeitslosenhilfe only available 1984 through 2005
+    out["arbeitslosenhilfe_y"] = object_to_int(raw_data["iunay"])
+    # grundsicherung only available 1984 through 2014
+    out["grundsicherung_y"] = object_to_int(raw_data["isuby"])
+
+    # private transfers contains
+    # alimony in 1984 through 2000
+    # divorce and caregiver alimonies in 1984 through 2014
+    # unterhaltsvorschuss in 1984 through 2009
+    out["private_transfers_received_amount_y"] = object_to_int(raw_data["ielse"])
+    # alimony received only available 2001 through 2014
+    out["alimony_received_amount_y"] = object_to_int(raw_data["ialim"])
+    # caregiver alimony received available since 2015
+    out["caregiver_alimony_received_amount_y"] = object_to_int(raw_data["ichsu"])
+    # divorce alimony only available in 2015
+    out["divorce_alimony_received_amount_y"] = object_to_int(raw_data["ispou"])
+    # unterhaltsvorschuss available since 2010
+    out["unterhaltsvorschuss_erhaltenener_betrag_y"] = object_to_int(raw_data["iachm"])
+    out["student_grants_received_amount_y"] = object_to_int(raw_data["istuy"])
```

Comment:
```suggestion
    out["stipendien_bafög_y"] = object_to_int(raw_data["istuy"])
```
Any of the two correct?